CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Flaky and Skip Changes to lookup.json
+The process for adding flaky or skip tags to the lookup.json is reasonably simple. Make your changes put together a PR with ideally a link to the community CI failing and then it will be reviewed.
+
+
+## Submitting a module to CitGM
+#### There are several requirements for adding a module to the lookup.json:
+
+#### Hard Requirements
+
+* Module source code must be on Github.
+* Published versions must include a tag on Github
+* The test process must be executable with only the commands
+`npm install && npm test` using the tarball downloaded from the Github tag
+mentioned above
+* The tests pass on supported major release lines
+* The maintainers of the module remain responsive when there are problems
+
+#### Soft Requirements
+
+At least one of:
+* The module must be actively used by the community
+OR
+* The module must be heavily depended on
+OR
+* The module must cover unique portions of our API
+OR
+* The module fits into a key category (e.g. Testing, Streams, Monitoring, etc.)
+OR
+* The module is under the Node.js foundation Github org
+OR
+* The module is identified as an important module by a Node.js Working Group
+
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
```

Comment:
I think general info should come first, and specific stuff like `Flaky and Skip Changes to lookup.json` should come afterwards.